Resources/Prototypes/Entities/Clothing/OuterClothing/base.yml

```diff
@@ -16,8 +16,10 @@
   name: base hardsuit
   components:
   - type: PressureProtection
-    highPressureMultiplier: 0.75
+    highPressureMultiplier: 0.5
     lowPressureMultiplier: 100
+  - type: TemperatureProtection
+    coefficient: 0.001 # yes it needs to be this low, fires are fucking deadly apparently!!!!
```

Comment:
If `TemperatureProtection` didn't prevent shivering/sweating, does this still have to be so high? I.e., is there a point where the sweating manages to negate the modified temperature increase from the fire?

I guess whenever hard suits get their own temperature regulation system (basically battery powered sweating/shivering), then this should probable be re examined?

Reply:
this will have to stay just because of how hot temperatures can get

Reply:
Also, while hard suits should obviously fully protect from cold, IMO a non-specialized hard suit shouldn't just make you **completely** immune to a large plasma fire.